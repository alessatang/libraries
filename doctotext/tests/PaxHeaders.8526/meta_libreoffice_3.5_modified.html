20 atime=1352037800
30 ctime=1391299512.896513768
