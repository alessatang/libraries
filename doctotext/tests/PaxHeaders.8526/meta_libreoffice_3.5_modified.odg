20 atime=1352037800
30 ctime=1391299512.807515304
