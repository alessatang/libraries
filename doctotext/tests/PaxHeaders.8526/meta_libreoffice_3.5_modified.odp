20 atime=1352037800
30 ctime=1391299512.793515545
