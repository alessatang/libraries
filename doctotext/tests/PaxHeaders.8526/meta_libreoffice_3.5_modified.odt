20 atime=1352037800
30 ctime=1391299512.769515959
