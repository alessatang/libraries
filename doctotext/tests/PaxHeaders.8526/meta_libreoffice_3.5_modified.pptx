20 atime=1352037800
30 ctime=1391299512.885513958
