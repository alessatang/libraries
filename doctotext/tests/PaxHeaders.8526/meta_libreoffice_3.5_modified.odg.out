30 mtime=1379553524.139591327
30 atime=1389151481.452853026
30 ctime=1389151481.452853026
